src/content/docs/fr/guides/integrations-guide/alpinejs.mdx

```diff
@@ -0,0 +1,129 @@
+  </Fragment>
+  <Fragment slot="yarn">
+    ```sh
+    yarn add @astrojs/alpinejs
+    ```
+  </Fragment>
+</PackageManagerTabs>
+
+La plupart des gestionnaires de paquets installent également les dépendances associées. Cependant, si vous voyez un avertissement "Cannot find package 'alpinejs'" (ou similaire) lorsque vous démarrez Astro, vous devrez installer manuellement Alpine.js vous-même :
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+    ```sh
+    npm install alpinejs @types/alpinejs
+    ```
+  </Fragment>
+  <Fragment slot="pnpm">
+    ```sh
+    pnpm add alpinejs @types/alpinejs
+    ```
+  </Fragment>
+  <Fragment slot="yarn">
+    ```sh
+    yarn add alpinejs @types/alpinejs
+    ```
+  </Fragment>
+</PackageManagerTabs>
+
+Ensuite, appliquez l'intégration à votre fichier `astro.config.*` en utilisant la propriété `integrations` :
+
+```js ins="alpine()" title="astro.config.mjs" ins={2}
+import { defineConfig } from 'astro/config';
+import alpine from '@astrojs/alpinejs';
+
+export default defineConfig({
+  // ...
+  integrations: [alpine()],
+});
+```
+
+## Utilisation
+
+Une fois l'intégration installée, vous pouvez utiliser les directives et la syntaxe [Alpine.js](https://alpinejs.dev/) dans n'importe quel composant Astro. Le script Alpine.js est automatiquement ajouté et activé sur chaque page de votre site web.
+
+Consultez notre [Documentation d'intégration Astro][astro-integration] pour plus d'informations sur les intégrations.
+
+## Limites
+
+L'intégration d'Alpine.js ne vous permet pas de contrôler la manière dont le script est chargé ou initialisé. Si vous avez besoin de ce contrôle, envisagez [d'installer et d'utiliser Alpine.js manuellement](https://alpinejs.dev/essentials/installation). Astro prend en charge toutes les instructions d'installation manuelle d'Alpine.js officiellement documentées, en utilisant les balises `<script>` à l'intérieur d'un composant Astro.
+
+**Il n'est actuellement pas possible d'[étendre Alpine.js](https://alpinejs.dev/advanced/extending) en utilisant ce composant.** Si vous avez besoin de cette fonctionnalité, envisagez de suivre [le manuel d'installation d'Alpine.js](https://alpinejs.dev/essentials/installation) au lieu d'utiliser une balise de script Astro :
+
+```astro title="src/pages/index.astro"
+---
+---
+
+<!-- Exemple : Charger AlpineJS sur une seule page. -->
+<script>
+  import Alpine from 'alpinejs';
+
+  // Optionnel: Étendre Alpine.js
+  // Alpine.directive('foo', ...)
+
+  window.Alpine = Alpine;
+  Alpine.start();
+</script>
+```
+
+## Configuration
+
+L'intégration d'Alpine.js ne prend pas en charge les configurations personnalisées pour le moment.
+
+## Exemples
+
+* L'exemple [Astro Alpine.js](https://github.com/withastro/astro/tree/latest/examples/framework-alpine) montre comment utiliser Alpine.js dans un projet Astro.
+
+[astro-integration] : /fr/guides/integrations-guide/
+
+[astro-ui-frameworks] : /fr/core-concepts/framework-components/#using-framework-components
```

Comment:
These lines seem out of place - are they intended to be here still?

Reply:
Hello! This is normal, in the English version, It's written the same way

Reply:
Yeah, this is normal Markdown syntax for "link shortcuts", just a note these can't have a space between the square brackets and `:`, so it just gets rendered as text instead.

Reply:
Ah, so it was the spaces then. Thanks Yan!